msg/Gpgst.msg

```diff
@@ -7,16 +7,16 @@ string message_id
 float64 utc_seconds
```

Comment:
I used float64 here because that was what was used by the GPGGA message, but shouldn't this be an integer? If it should, we can leave it as a float here to be consistent with the other messages, or I can change the other messages to be integers also.

Reply:
This can include decimal seconds up to two decimal places. Float32 would arguably be precise enough since the range is only [0.00, 86400.00), but I don't want to change the existing message and break backwards compatibility, so I think float64 is the right choice here.